testing/MLDB-1155_csv_line_endings.py

```diff
@@ -199,7 +197,7 @@
     assert json.loads(result['response'])[1][2] == 1
 
 
-    # CSV importation should accept a blank trailing line with DOS
+    # CSV importation should accept a blank trailing line with DOS if ignoreBadLines is set to true
```

Comment:
It should accept a blank trailing line no matter what ignoreBadLines is set to; it's too common for us tom consider that a bad line.